holmes/core/toolset_manager.py

```
@@ -113,14 +114,27 @@ def _list_all_toolsets(
+        self.check_toolset_prerequisites(enabled_toolsets)
 
         return list(toolsets_by_name.values())
 
+    @classmethod
+    def check_toolset_prerequisites(cls, toolsets: list[Toolset]):
+        with concurrent.futures.ThreadPoolExecutor() as executor:
+            futures = []
+            for toolset in toolsets:
+                futures.append(executor.submit(toolset.check_prerequisites))
+
+            for future in concurrent.futures.as_completed(futures):
```

Ruff (0.12.2)

Loop control variable `future` not used within loop body

Rename unused `future` to `_future`

(B007)